helper-cli/src/main/kotlin/commands/GenerateTimeoutErrorResolutionsCommand.kt

```diff
@@ -39,34 +39,40 @@ import java.io.File
 
 @Parameters(
     commandNames = ["generate-timeout-error-resolutions"],
-    commandDescription = "Generates resolutions for scanner timeout errors."
+    commandDescription = "Generates resolutions for scanner timeout errors. The result is written to the standard " +
```

Comment:
Nit: You're not only adding description but also improving an existing description.